api/bazel/repository_locations.bzl

```diff
@@ -39,11 +39,11 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         project_desc = "xDS API Working Group (xDS-WG)",
         project_url = "https://github.com/cncf/xds",
         # During the UDPA -> xDS migration, we aren't working with releases.
-        version = "e9ce68804cb4e64cab5a52e3c8baf840d4ff87b7",
-        sha256 = "0d33b83f8c6368954e72e7785539f0d272a8aba2f6e2e336ed15fd1514bc9899",
+        version = "expression-new-fields",
```

Comment:
Let me know when ready for review. It looks like this is still pointing at a branch.

Reply:
Yes - this is still a draft. I'll click "Ready for review" when it's ready.